processing/src/test/java/io/druid/segment/filter/BaseFilterTest.java

```diff
@@ -302,7 +302,7 @@ private DimFilter maybeOptimize(final DimFilter dimFilter)
     return adapter.makeCursors(
         filter,
         new Interval(JodaUtils.MIN_INSTANT, JodaUtils.MAX_INSTANT),
-        VirtualColumns.EMPTY,
```

Comment:
i think you are commenting on the change before the merge. this file had a merge conflict with the one from master. i restored it.